v2/spanner-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/exceptions/InvalidSourceException.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.templates.exceptions;
+
+/** Exception in case of unsupported source database. */
+public class InvalidSourceException extends Exception {
```

Comment:
nit: Maybe just call it UnsuportedSourceException as that is what it is meant to be.